package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "to-do-list-app",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "^6.0.3",
```

Comment:
Please use strict package version like: "@angular/animations": "6.0.3" - without any additional symbols, also please take a look at this post to clarify what they mean - https://docs.npmjs.com/files/package.json#dependencies